media/css/firefox/firstrun/firstrun_quantum.less

```diff
@@ -0,0 +1,542 @@
+@import "../../sandstone/lib.less";
+
+@font-face {
+    font-family: 'Fira Sans Light';
+    src: url('/media/fonts/FiraSans-Light.woff2') format('woff2'),
+         url('/media/fonts/FiraSans-Light.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Fira Sans Light';
+    src: url('/media/fonts/FiraSans-SemiBold.woff2') format('woff2'),
+         url('/media/fonts/FiraSans-SemiBold.woff') format('woff');
+    font-weight: bold;
+    font-style: normal;
+}
+
+:root {
+  --animation-curve: linear;
+  // --animation-curve: cubic-bezier(.73,.08,.94,.33);
+  // --backwards: cubic-bezier(.49,-0.41,.96,.67);
+}
+
+body {
+    margin: 0;
+    overflow: hidden;
+}
+
+* {
+    font-family: 'Fira Sans Light';
+}
+
+#scene {
+    height: 100vh;
+    width: 100vw;
+}
+
+.background {
+    transform: scale(1.5);
+    position: absolute;
+    right: 0;
+    left: 0;
+    margin: auto;
+}
+
+.white-overlay {
+    fill: #FFFFFF;
+    height: 100vh;
+    width: 100vw;
+    position: absolute;
+    opacity: 0;
+}
+
+.st1 {
+    fill: url('#SVGID_1');
+    height: 100vh;
+    width: 100vw;
+}
+
+.st2 {
+    fill: url('#SVGID_2');
+    height: 70vh;
+    width: 85vw;
+    bottom: 0;
+}
+
+.st3 {
+    opacity: 0.5;
+    fill: url('#SVGID_2');
+    height: 48vh;
+    width: 75vw;
+    bottom: 0;
+}
+
+.st4 {
+    opacity: 0.6;
+    fill: #005FE7;
+    height: 26vh;
+    width: 100vw;
+    bottom: 0;
+}
+
+.st5 {
+    opacity: 0.6;
+    fill: #002079;
+    height: 51vh;
+    width: 100vw;
+}
+
+.st6 {
+    opacity: 0.8;
+    fill: url('#SVGID_1');
+    height: 73vh;
+    width: 100vw;
+}
+
+.st7 {
+    opacity: 0.3;
+    fill: url('#SVGID_1');
+    margin: 0;
+    height: 59vh;
+    width: 60vw;
+}
+
+.st8 {
+    opacity: 0.6;
+    fill: #002079;
+    height: 51vh;
+    width: 100vw;
+}
+
+.cls-2 {
+    height: 100vh;
+    width: 100vw;
+    position: absolute;
+    fill:url('#linear-gradient');
+    mix-blend-mode:color;
+    opacity: 0;
+}
+
+#firefox-logo {
+    background: url('/media/img/firefox/firstrun-quantum/quantum-logo.svg');
+    width: 80px;
+    height: 80px;
+    background-size: 100%;
+    background-repeat: no-repeat;
+    opacity: 0;
+    position: relative;
+    float: left;
+    clear: both;
+}
+
+#fxa {
+    border: none;
+    position: relative;
+    width: 420px;
+}
+
+#fxa-iframe-config {
+    background: #fff;
+    display: inline-block;
+    margin-left: 77px;
+    transform: scale(.8);
+    opacity: 0;
+    padding-bottom: 20px;
+}
+
+.fxaccounts-container {
+    bottom: 0;
+    color: #fff;
+    height: 485px;
+    left: 0;
+    margin: auto;
+    position: absolute;
+    right: 0;
+    top: 0;
+    width: 980px;
```

Comment:
This fixed width seems a bit too wide for safety and might assume larger windows than some people will have. We don't need to be too concerned about really small viewports since this isn't shown on mobile, but we could add a media query or two to make things more compact for smaller windows. The old page just fits at 750 x 500 so that seems like a decent minimum size to aim for. We could go fully responsive and stack boxes in really skinny windows but that might be overkill.